doc/tech/approach.md

```diff
@@ -35,9 +35,11 @@ For CNTT, a strategy is needed that affords Operators the performance, flexibili
 
 
 <a name="5.1.1"></a>
-### 5.1.1 Executive Summary
+### 5.1.1 Networking Scope
+According to the ETSI NFV Model, Networking alongside Compute and Storage, is an integral part of the Network Function Virtualisation Infrastructure (NFVI). The general function of Networking in the NFV context is to provide the connectivity between various virtualised and non-virtualised resources required for a delivery of a Network Service. Such a connectivity may manifest itself as a virtualised network between VMs or Containers (e.g. overlay networks managed by SDN Controllers, or programmable fabrics that provide such connectivity natively) or as an integration into the Infrastructure Hardware level for offloading some of the Network Functions into this level. 
+
+The main concern of this specification, is the normalization of the integration reference points between different layers of the NFV architecture stack.  In the Networking context the primary focus is directed on the packet flow and control flow interfaces between Virtualization level (be it IaaS or CaaS type of virtualisation) and Networking Hardware resources, as well as on related integration into MANO level (Hardware/ Network Infrastructure Management/Control, Orchestration).  The Networking Sope includes hence a definition of a normalized Abstraction layer between Virtualization Layer domains and the Networking Hardware layer resource pool in a way that the implementation details of the Networking Hardware are not visible to the VNFs/CNFs, and preferably not even visible to the Virtualization layer.
```

Comment:
"The Networking **_Sope_** includes hence a definition of a normalized Abstraction layer between Virtualization Layer domains and the Networking Hardware layer resource pool in a way that the implementation details of the Networking Hardware are not visible to the VNFs/CNFs, and preferably not even visible to the Virtualization layer."

Typo: "Sope"
Sentence: ""The Networking Sope **_includes hence_** a definition of a ..."
Standardise use of "layer" -- upper case "L", lower case "l"

Given the use of the term "Virtualization Layer" the same term should be used in the previous sentence instead of "Virtualization Level."

Reply:
@pgoyal01 Thanks for your comments but the current text does not contain these errors. You must have looked at an old text?